### PR TITLE
zh-cn: update the translation of "Set-Cookie"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of "Set-Cookie"

### CODE REVIEW

File: files/zh-cn/web/http/headers/set-cookie/index.md [hunk 1]
```diff
@@ -22,178 +24,188 @@ slug: Web/HTTP/Headers/Set-Cookie
...
-- `Domain=<domain-value>` {{optional_inline}}
+    多个主机/域名的值是*不*被允许的，但如果*指定*了一个域名，则其子域名也总会被包含。
 
-  - : 指定 cookie 可以送达的主机名。
-
-    假如没有指定，那么默认值为当前文档访问地址中的主机部分（但是不包含子域名）。
+- `Expires=<date>` {{optional_inline}}
 
-    与之前的规范不同的是，域名（`.example.com`）之前的点号会被忽略。
+  - : cookie 的最长有效时间，以 HTTP 日期时间戳的形式。参见 {{HTTPHeader("Date")}} 以了解要求的格式。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/set-cookie/index.md:85]:
```suggestion
  - : 以 HTTP 日期时间戳形式指定的 cookie 的最长有效时间。参见 {{HTTPHeader("Date")}} 以了解要求的格式。
```

File: files/zh-cn/web/http/headers/set-cookie/index.md [hunk 2]
```diff
@@ -22,178 +24,188 @@ slug: Web/HTTP/Headers/Set-Cookie
...
 
-  - : 指定一个 URL 路径，这个路径*必须*出现在要请求的资源的路径中才可以发送 `Cookie` 标头。
+    如果设置了 `Expires` 日期，其截止时间与*客户端*相关，而非服务器的时间。
 
-    字符 `/` 可以解释为文件目录分隔符，此目录的下级目录也满足匹配的条件（例如，如果 `path=/docs`，那么
-
-    - `/docs`、`/docs/`、`/docs/Web/` 和 `/docs/Web/HTTP` 都满足匹配条件。
-    - `/`、`/docsets` 或者 `/fr/docs` 则不满足匹配条件。
+- `HttpOnly` {{optional_inline}}
+  - : 阻止 JavaScript 通过 {{domxref("Document.cookie")}} 属性访问 cookie。注意，设置了 `HttpOnly` 的 cookie 在 JavaScript 初始化的请求中仍然会被发送。例如，调用 {{domxref("XMLHttpRequest.send()")}} 或 {{domxref("fetch()")}}。其用于防范跨站脚本攻击（{{Glossary("Cross-site_scripting", "XSS")}}）。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/set-cookie/index.md:94]:
```suggestion
  - : 阻止 JavaScript 通过 {{domxref("Document.cookie")}} 属性访问 cookie。注意，设置了 `HttpOnly` 的 cookie 仍然会通过 JavaScript 发起的请求发送。例如，调用 {{domxref("XMLHttpRequest.send()")}} 或 {{domxref("fetch()")}}。其用于防范跨站脚本攻击（{{Glossary("Cross-site_scripting", "XSS")}}）。
```

File: files/zh-cn/web/http/headers/set-cookie/index.md [hunk 3]
```diff
@@ -22,178 +24,188 @@ slug: Web/HTTP/Headers/Set-Cookie
...
+  - : 表示仅当请求通过 `https:` 协议（localhost 不受此限制）发送时才会将该 cookie 发送到服务器，因此其更能够抵抗[中间人](/zh-CN/docs/Glossary/MitM)攻击。
 
-      > **备注：** 与 [SameSite Cookie](/zh-CN/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value) 相关的标准作了如下变动：
-      >
-      > 1. `SameSite` 属性未被指定时，其默认行为是 `SameSite=Lax`。在过去，若未指定，所有的 cookie 均会被发送。
-      > 2. 现在，携带 `SameSite=None` 属性的 cookie 必须同时设置 `Secure` 属性（换句话说，其仅能用于安全上下文）
-      > 3. 来自同一域的 cookie 若使用了不同的协议（`http:` 或 `HTTPS:`），将不再被视为来自同一站点。
-      >
-      > 参见[浏览器兼容性](/zh-CN/docs/Web/HTTP/Headers/Set-Cookie#浏览器兼容性)表以了解不同浏览器的实线情况（行：“`SameSite`: Defaults to `Lax`”、“`SameSite`: Secure context required”和“`SameSite`: URL scheme-aware ("schemeful")”）。
+    > **备注：** 不要假设 `Secure` 会阻止所有的对 cookie 中敏感信息（会话密钥、登录信息，等等）的访问。携带这一属性的 cookie 在不设置 `HttpOnly` 属性的情况下仍能从客户端的硬盘或是从 JavaScript 中访问。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/set-cookie/index.md:134]:
```suggestion
    > **备注：** 不要假设 `Secure` 会阻止所有的对 cookie 中敏感信息（会话密钥、登录信息，等等）的访问。携带这一属性的 cookie 在不设置 `HttpOnly` 属性的情况下仍能从客户端的硬盘或是从 JavaScript 中访问及更改。
```

File: files/zh-cn/web/http/headers/set-cookie/index.md [hunk 4]
```diff
@@ -22,178 +24,188 @@ slug: Web/HTTP/Headers/Set-Cookie
...
 Set-Cookie: <cookie-name>=<cookie-value>; SameSite=Strict
 Set-Cookie: <cookie-name>=<cookie-value>; SameSite=Lax
 Set-Cookie: <cookie-name>=<cookie-value>; SameSite=None; Secure
 
-// Multiple attributes are also possible, for example:
+// 可以同时有多个属性，例如：
 Set-Cookie: <cookie-name>=<cookie-value>; Domain=<domain-value>; Secure; HttpOnly
 ```
 
 ## 指令
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/set-cookie/index.md:53]: Should we change original document to 'Directives' here?

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/http/headers/set-cookie/index.md (hunk 4 above)
Comment (yin1999) [reply files/zh-cn/web/http/headers/set-cookie/index.md:53]: Actually, we could use "attributes", as this is used by the [spec](https://httpwg.org/specs/rfc6265.html#sane-set-cookie).

```suggestion
## 属性
```

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**